### ISSUE TITLE
Wrong 3D file in scene 10: dog

### ISSUE THREAD

**Issue by johanne-aarup-hansen** [mirroringplaces/mirroringplaces.online#101]
It is the wooden statues

**Comment by johanne-aarup-hansen** [comment]
There are three chapters that are missing or displaying wrong objects:

- Chapter 8 misses a stone
- Chapter 10 display wooden sculptures and not the dog file
- Chapter 14.´misses the computer board file
I dont understand why, because it looks like all files are there... do you know Balder?

**Comment by johanne-aarup-hansen** [comment]
Hey Balder
Nu har jeg de færdige filer, men hvor skal jeg uploade dem til?

**Comment by benjaoming** [comment]
Hej! De ligger her: https://github.com/mirroringplaces/mirroringplaces.online/tree/main/static/3dmodels

Du kan erstatte de gamle filer og gebruge deres navne. Så burde det ikke være nødvendigt at redigere yderligere.

**Comment by johanne-aarup-hansen** [comment]
Så nu har jeg oploadet de nye filer, lavevt pull request og sån, men jeg kan altså ikke se forskel. Fx. så viser scene 10, som skal være med en hund, stadig de små træfigure. Vil du kigge på det Balder?

**Comment by benjaoming** [comment]
Ser det korrekt ud nu?

**Comment by johanne-aarup-hansen** [comment]
Nu virker scene 10 med hunden!! De filer udskriftede jeg også, men kan se de andre filer jeg har uploadet ikke virker. Det drejer sig om scene 5, 8, 9 og 14

**Comment by benjaoming** [comment]
Hvis vi tager f.eks. scene 5, så er .gtlf filen forkert nu:

https://github.com/mirroringplaces/mirroringplaces.online/blob/32fba2bf87d5d5acb963698d790fe54c48d01b69/static/3dmodels/5_bottle_opener/5_bottle_opener.gltf#L146-L151

Den skal pege på korrekt filnavn, 5_bottle_opener.bin

Se også kommentar her: jeg har manuelt rettet stort set alle .gltf filerne, fordi de indeholder forkerte filnavne... jeg ved ik hvad årsagen kan være.

**Comment by benjaoming** [comment]
Jeg var nødt til manuelt at rette stort set alle .gtlf-filer, jeg ved ikke hvor fejlen opstår; men det har hele tiden været sådan at .gtlf-filerne peger på ikke-eksisterende .bin-filer.

**Comment by benjaoming** [comment]
For at få scene 5, 8, 9 og 14 til at virke igen, skal du have rettet .gtlf-filerne ved at åbne dem i en tekst-editor

Jeg ville også foreslå at køre `hugo server` lokalt for at teste tingene, inden de bliver lagt op :innocent:

**Comment by johanne-aarup-hansen** [comment]
Nu har jeg ændret bin navnet i glt filen på alle filer... men der er stadig tre filer som ikke virker... Det drejer sig nu om 8, 9 og 14.
Kan du komme på andre mulige problemr jeg skal rette?

**Comment by benjaoming** [comment]
Ja, udover .bin-filerne, så skal jpg-filerne også stå skrevet korrekt i .gflt-filerne.

F.eks. i scene 8, står:

https://github.com/mirroringplaces/mirroringplaces.online/blob/f55297bd844e3779b71505eeb348e07d277c8bb1/static/3dmodels/8_stone/8_stone.gltf#L238-L244

**Comment by benjaoming** [comment]
Jeg tror at alle .jpg-filer hedder `texture.jpg`.
